pubtools/_quay/signature_handler.py

```diff
@@ -488,7 +550,10 @@ def sign_container_images(self, push_items):
 
         claim_messages = []
         for item in push_items:
-            claim_messages += self.construct_item_claim_messages(item)
+            if only_v2s1_manifests:
```

Comment:
related to my comment in pubtools/_quay/push_docker.py - can we construct here both digest everytime? Existing manifest claim will be filtered out by filter_claim_messages

Reply:
What comment?

Reply:
That's strange. I added it there now